README.md

```diff
@@ -34,6 +34,7 @@ Configures Logical Volume Groups, Logical Volumes, Filesystems, mount points, an
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`vg`: Volume Group to create the Logical Volume in   
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`lv`: Name of the Logical Volume to create   
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`size`: Size of the Logical Volume to create   
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`opts`: Free-form options to be passed to the lvcreate command  
```

Comment:
Should we call it "lvol_opts" in case more such options come (and this is passed to lvcreate yet through lvol Ansible module) ?

Edit: perhaps not if that's specific ONLY to the lv entries; I would add this option to the example though.

Reply:
Good idea - https://github.com/speechmatics/ansible-lvm/pull/4/commits/beca9f0f5bcb6c16bed4466b87a04d306a33828c adds it to the example playbook in README

I've left it as `opts` for now  since yeah, it's a freetext field that can take multiple options just delimited by a space - I realise it probably wasn't clear without the example though!